### PR TITLE
Temp dev

### PR DESCRIPTION
**Pull request by deekshithmd**
Added responsiveness to ecommerce site, please review it and provide your comments

### CODE REVIEW

File: src/components/banner/banner.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
+.product-banner {
+    position: relative;
+    width: 100%;
+    max-height: 60vh;
+    margin: 30px auto;
```
Comment (Ananyamadhu08) [inline src/components/banner/banner.css]: can use rem instead of px for higher values.

**Review by Ananyamadhu08 — APPROVED** [review]
Hey Deekshith. Your code is neat and understandable. I like the dark mode feature in your app, you could make the search-bar text white in dark mode for readability. You can go ahead and merge your PR.

File: src/components/banner/banner.css (hunk 1 above)
Comment (deekshithmd) [reply src/components/banner/banner.css]: Sure, thank you for pointing it out

**Comment by deekshithmd** [comment]
> Hey Deekshith. Your code is neat and understandable. I like the dark mode feature in your app, you could make the search-bar text white in dark mode for readability. You can go ahead and merge your PR.

Ok, I will check this.

**Review by amartya-rs — APPROVED** [review]
Hey Deekshit,
Your code looks fine to me. Good to merge.

**Review by saurabhchirde — APPROVED** [review]
Clean and easy to read code, Looks good to merge.

**Pull request merged**